GameData/RP-0/Strategies/Leaders/LeadersEngineers.cfg

```diff
@@ -389,3 +389,59 @@ STRATEGY
 		}
 	}
 }
+
+STRATEGY
+{
+	name = leaderValCleaverEngineer
+	RP0conf = True
+	title = Arthur Valentine Cleaver
+	desc = Arthur "Val Cleaver" is a British Aerospace Engineer working for de Havilland in the early 1950s, responsible for overseeing the development of the early British rocket plane engines, "Spectre" and "Sprite". Later he worked to develop upgraded variants of the "Spectre" engine, that went on to power the Blue Steel missile.  In 1956, Val Cleaver hands in his notice to de Havilland, and takes a job with Rolls-Royce as their Chief Rocket Propulsion Engineer. Under his guidance the development of the RZ.2 and RZ.20 takes place, built to power the Blue Streak missile. Considered highly advanced engines at the time, it later goes on to power the first stage of Europa 1. Arthur Val Cleaver is awarded an Order of the British Empire, a prestigious medal for prominent national achievements.
+	department = Engineering
+	icon = RP-0/Strategies/Leaders/ValCleaver
+	iconDepartment = RP-0/Strategies/Leaders/ValCleaver
+
+
+	groupTag = Engineering,leaderValCleaver
+
+	removeOnDeactivate = True
+	removeOnDeactivateTag = leaderValCleaver
+  
+	minDuration = 2592000 // 30 days
+	maxDuration = 315576000 // 10 years
+	reactivateCooldown = 31557600 // 1 year
+	
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = Rolls-RoyceDidRocketsGood!
+		multiplier = 1.05
+		effectDescription = Early and Orbital Rocket Engine tech
+		nodeTypes
+		{
+			typeNode = RocketEngines
+		}
+	}
+
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = BritishRocketPlanes
+		multiplier = 1.05
+		effectDescription = Flight tech
```

Comment:
I'm a little skeptical of this one as well, as we have so many other things that give a bonus to flight research. Can you think of any other advantages?

Reply:
I was hoping a splash of British was unique enough, similar enough in rocketry and planes research that it would only be used for A Very British Space Program type deal. Players self made house rules preventing use of American or Soviet nationals ect. (I can't be the only one who does it! aha)
Perhaps Kerolox bonuses as an alternative? I will try figure out the keywords for that

Reply:
We've tried to make sure no leaders duplicate other leaders, and it would be really good for that to continue. Otherwise adding this leader adds no gameplay and could be better served by some cosmetic mod that just changes names and faces. There's a ton of keywords (in technodedata and in KCTTags.cfg and in CurrencyModifierRP0.cs) so there should be plenty of options